chapters/ch37.md

```diff
@@ -1,71 +1,70 @@
 > # 37. How to decide whether to use all your data
 
---> _replace THIS LINE by your translation for the above line_
+Làm sao để quyết định có nên sử dụng toàn bộ dữ liệu của bạn hay không?
 
 > Suppose your cat detector’s training set includes 10,000 user-uploaded images. This data comes from the same distribution as a separate dev/test set, and represents the distribution you care about doing well on. You also have an additional 20,000 images downloaded from the internet. Should you provide all 20,000+10,000=30,000 images to your learning algorithm as its training set, or discard the 20,000 internet images for fear of it biasing your learning algorithm?
 
---> _replace THIS LINE by your translation for the above line_
+Giả sử tập huấn luyện của bộ nhận diện mèo của bạn bao gồm 10.000 hình ảnh do người dùng tải lên. Dữ liệu này đến từ cùng một phân phối chia ra một tập phát triển/kiểm tra riêng biệt, và đại diện cho phân phối mà bạn muốn làm tốt trên đó. Bạn cũng có thêm 20.000 hình ảnh được tải xuống từ internet. Bạn có nên cung cấp tất cả 20.000 + 10.000 = 30.000 hình ảnh cho thuật toán learning của mình dưới dạng tập huấn luyện của nó, hay nên loại bỏ 20.000 hình ảnh từ internet vì sợ nó thiên vị thuật toán learning của bạn?
 
 
 > When using earlier generations of learning algorithms (such as hand-designed computer vision features, followed by a simple linear classifier) there was a real risk that merging both types of data would cause you to perform worse. Thus, some engineers will warn you against including the 20,000 internet images.
 
---> _replace THIS LINE by your translation for the above line_
+Khi sử dụng các thế hệ thuật toán learning trước đó (như các tính năng nhận diện hình ảnh được thiết kế thủ công, theo sau là một phân loại tuyến tính đơn giản), có một rủi ro thực sự là việc hợp nhất cả hai loại dữ liệu sẽ khiến bạn đạt hiệu quả tệ hơn. Do đó, một số kỹ sư sẽ cảnh báo bạn không bao gồm 20.000 hình ảnh internet.
```

Comment:
>Khi sử dụng các thế hệ thuật toán learning trước đó (như các tính năng nhận diện hình ảnh được thiết kế thủ công, theo sau là một phân loại tuyến tính đơn giản), có một rủi ro thực sự là việc hợp nhất cả hai loại dữ liệu sẽ khiến ~bạn đạt hiệu quả~ chất lượng thuật toán tệ hơn. Do đó, một số kỹ sư sẽ cảnh báo bạn không ~bao gồm~ thêm vào 20.000 ~hình~ ảnh internet.

Reply:
chỗ "cause you to perform worse" mình nghĩ không liên quan tới thuật toán mà gần với hiệu quả của model nên mình nghĩ không sửa chỗ này nha bạn.
2 points còn lại mình đã edit. Cảm ơn bạn.